oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-cluster.json

```diff
@@ -1,19 +1,3 @@
-/**
```

Comment:
@kezhenxu94 Should we keep the license header in the UI template files?

Reply:
I think we no longer need these headers: https://github.com/apache/skywalking/pull/11184#discussion_r1285683989

Reply:
@smartboy37597 Could you follow up on another PR to remove other existing headers to keep them consistent?

Reply:
> @smartboy37597 Could you follow up on another PR to remove other existing headers to keep them consistent?

Okey, I'm going to delete all UI headers in a new PR.